src/global/Layouts/StickyNavigation/NavigationElement/NavigationElement.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Icon } from 'components/Icon';
+import { Col } from 'antd';
+import { Link } from 'react-router-dom';
+
+type NavActive = {
+  isActive: boolean;
+};
+
+const NavIcon = styled(Icon)<NavActive>`
+  background-color: ${props =>
+    props.isActive ? props.theme.colors.main.primary : props.theme.colors.functional.disabled};
+`;
+
+const CenteredCol = styled(Col)`
+  && {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+`;
+
+const NavText = styled.span.attrs({
+  style: (props: any) => props.theme.typography.subtitle2,
```

Comment:
Because it doesn't recognize themes props here. I don't want to add ThemeProps anytime I will use attrs - we should use some generic solution to tell it once and globally, but right now I had no time for this. If you have better solution then let me know.